src/bun.js/api/bun/subprocess.zig

```diff
@@ -2640,6 +2676,7 @@ pub const Subprocess = struct {
             .stdout = Readable.init(stdio[1], bun.toFD(stdout_pipe[0]), jsc_vm.allocator, default_max_buffer_size),
             .stderr = Readable.init(stdio[2], bun.toFD(stderr_pipe[0]), jsc_vm.allocator, default_max_buffer_size),
             .stdio_pipes = stdio_pipes,
+            .stdio_inherits = stdio_inherits,
             .on_exit_callback = if (on_exit_callback != .zero) JSC.Strong.create(on_exit_callback, globalThis) else .{},
```

Comment:
I think we need to add `.stdio_close = stdio_close` here, as well?

Reply:
/facepalm, yes, good catch, thanks. Pushed an update for that. (The default initializers on these structs aren't being very helpful in this case.)